mackup/applications/intellijidea-2016-1.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = IntelliJ IDEA 2016.1
+
+[configuration_files]
+Library/Application Support/IntelliJIdea2016.1
+Library/Preferences/IntelliJIdea2016.1
```

Comment:
The Community version name not include `IntelliJ`

    Library/Application Support/IdeaIC2016.1
    Library/Preferences/IdeaIC2016.1